__tests__/integration/validationErrors.test.js

```diff
@@ -0,0 +1,29 @@
+// Copyright 2019 Stanford University see LICENSE for license
+
+import pupExpect from 'expect-puppeteer'
+import { testUserLogin } from './loginHelper'
+import { incompleteFieldsForBibframeInstance } from './previewRDFHelper'
+
+describe('Validation errors', () => {
+  beforeAll(async () => {
+    await testUserLogin()
+    await incompleteFieldsForBibframeInstance()
+  })
+
+  it('makes a change in resource', async () => {
+    expect.assertions(2) // +1 from incompleteFieldsForBibframeInstance
+
+    // Click on the PreviewRDF button and a modal appears
+    await pupExpect(page).toClick('button.btn-remove[data-id=\'content\']', { text: 'Remove' })
```

Comment:
after clicking this button, should there be some test?  Maybe this test should be folded into the case below?

Reply:
Fixed.